api/entity_config/config.go

```diff
@@ -12,6 +12,12 @@ func ConfigValue(network *net.Network, application, entity, config string) (stri
 	if nil != err {
 		return "", err
 	}
+
+	var value string;
+	err = json.Unmarshal(bytes, &value)
+	if nil == err {
```

Comment:
I'm not convinced line 21 is a happy path - you tried to unmarshal a string, but got something else unexpected.
I was thinking
```
    if nil != err {
        return string(bytes), err
    }
    return value, nil
```
i.e return the `err` too because calling code might want to know why the unmarshal failed.